src/otomi.ts

```diff
@@ -8,18 +10,13 @@ import { basicOptions } from './common/yargs-opts'
 console.profile('otomi')
 const debug = terminal('global')
 const terminalScale = 0.75
-const isAutoCompletion = process.argv.includes('--get-yargs-completions')
-if (!env.IN_DOCKER && !isAutoCompletion) {
-  debug.error(process.argv)
-  debug.error('Please run this script using the `otomi` entry script')
-  process.exit(1)
-}
-if (env.TESTING) {
-  process.env.AZURE_CLIENT_ID = 'somevalue'
-  process.env.AZURE_CLIENT_SECRET = 'somesecret'
-}
 
 const startup = async (): Promise<void> => {
+  const link = `${process.cwd()}/env`
```

Comment:
I am not against this logic but the naming `link` is confusing and ambiguous. I am just trying to gain the perspective of someone who is unfamiliar with the code.

Reply:
Does its use not help clarify? It is obviously used to create a symbolic link.

Reply:
I know that is a symbolic link, but that is not a good way to describe a variable. It is the same as saying that you call a variable "var" because it is a variable. You should call this variable something that says something about how or why the symbolic link exists.

Reply:
That is definitely not the same. I did not call it "var" now did I? I gave it a short and meaningful name. Please don't waste cycles on this

Reply:
`symlinkEnvDir`